### ISSUE TITLE
Improve support script to debug connection and latency issues

### ISSUE THREAD

**Issue by mogren** [aws/amazon-vpc-cni-k8s#807]
We need more data gathered to figure out timing issues. For example iptables version, kernel version, CoreDNS version, test for Calico, check for musl. Retransmission counters via BPF. IPv6 problems, make sure disable it.

**Comment by nithu0115** [comment]
@mogren, I have written a log collector which collects a bit of information (like containers logs, messages, sysctl output). We can leverage existing script and build on top of it. 

https://github.com/awslabs/amazon-eks-ami/tree/master/log-collector-script/linux

Is that something we are trying to achieve here ?

**Comment by mogren** [comment]
Yes, that is the idea. Improve that script, and import it into this repository whenever it's updated.

**Comment by nithu0115** [comment]
#803
